data/api_v1.go

```diff
@@ -93,6 +93,7 @@ func (a *API) setHandlesV1(prefix string, rtr *mux.Router) {
 	rtr.HandleFunc(prefix+"/summary/{userID}", a.middlewareV1(a.getDataSummaryV1, true, "userID")).Methods("GET")
 	rtr.HandleFunc(prefix+"/data/{userID}", a.middlewareV1(a.getDataV1, true, "userID")).Methods("GET")
 	rtr.HandleFunc(prefix+"/{.*}", a.middlewareV1(a.getNotFoundV1, false)).Methods("GET")
+	rtr.HandleFunc(prefix+"/imei/{imei}", a.middlewareV1(a.getImeiV1, false, "imei")).Methods("GET")
```

Comment:
Anyone with a valid token can access this route, I think we should restrict it to server tokens (which involves to change the middlewareV1 that doesn't handle for now server only authorizations).
 
Also If we want to set this route on private ingress only, it would be simpler if we had a different prefix (or we will have to manually list all authorized prefixes in public ingress config i.e /v1/(data|summary|range) )